### ISSUE TITLE
Good morning?

### ISSUE THREAD

**Issue by cjssh1002** [Aunsiels/alphazero_janggi#1]
Hi, I am interested in your project.
You are not Korean, how did you plan to develop janggi?

Can you elaborate on your project?

I just want us to share some opinions on janggi theory.

**Comment by Aunsiels** [comment]
Hi!

It is nice to see you are interested in this project! Indeed, I am not Korean, but I am working with a friend who is a professional Janggi player: https://www.youtube.com/channel/UCksfz2MkgxAKNJDvzemO3iA .

I am currently training neural networks on GPUs. I did the first try with 20 residual blocks, and I am now trying with 40, like in the original paper. I will put on Github the best network I got so far, but it did not reach a level good enough. It still knows some openings.

On the first try, I pretrained the network with existing games (supervised learning), but I had the feeling that it makes the network learn only one good move, and thus the exploration is not that good. Now I am learning without existing games.

The main problem is that I do not have enough computational power, so it takes time and I cannot really finetune the learning parameters.

We are also developing other tools for Janggi if you are interested!

**Comment by cjssh1002** [comment]
Nice to meet you.

Currently, I am very interested in janggi AI research, and I am analyzing janggi using an AI program. In addition, I asked some developer to develop the janggi program. (Because I am not a professional chess programmer)
There is a case of someone trying to develop janggi with AlphaZero in a similar way in Korea. However, he did not achieve satisfactory results.
It takes a lot of time and hardware resources and, as you said, struggled with fine-tuning the evaluation.

I have some questions.
There are some janggi data that I have studied. (specific patterns for janggi and endgame win-loss data)
Is this information help improve AI strength?
how to run your program on Windows 10 system? (with gui)

**Comment by Aunsiels** [comment]
For your questions:

Real games are useful in the case of supervised learning. I already have some https://github.com/Aunsiels/alphazero_janggi/tree/main/data, but I did not find many. So, if you have more, it would be great to add them and see if it helps.
As we do not speak Korean, it is hard for us to find resources online.

For Windows 10, I have not tried yet. I will see if I can program a small GUI to test the program.

**Comment by cjssh1002** [comment]
ok,
I will provide the data for learning.
Let me know your email address.

**Comment by cjssh1002** [comment]
ok,
I will provide the data for learning.
Let me know your email address.
